packages/server-wallet/src/models/challenge-status.ts

```diff
@@ -1,12 +1,18 @@
+import {State} from '@statechannels/wallet-core';
 import Knex from 'knex';
 import _ from 'lodash';
 import {Model} from 'objection';
 
 import {Bytes32, Uint48} from '../type-aliases';
 
 export type ChallengeStatusResult =
```

Comment:
I thought that this table/model was also going to be used to when a `Conclude` transaction succeeded. Is there a reason why it isn't called something like `AdjudicatorStatusResult`?

Reply:
I think something like `AdjudicatorStatusResult` makes sense. Originally I wasn't considering `Conclude`.